docs/configuration/backends/file.md

```diff
@@ -8,6 +8,8 @@ You have three choices:
 - [Rules in a Separate File](/configuration/backends/file/#rules-in-a-separate-file)
 - [Multiple `.toml` Files](/configuration/backends/file/#multiple-toml-files)
 
+To enable the file backend, you must either pass `--file` option to Træfik binary or put `[file]` section (with or without inner settings) in the configuration file.
```

Comment:
_[...] either pass `--file` option to Traefik binary or put `[file]` section [...]_

 ->

_[...] either pass **the** `--file` option to **the** Traefik binary or put **the** `[file]` section [...]_

(**Emphasis** by me.)